src/index.js

```diff
@@ -1,5 +1,5 @@
-import redata from './redata';
+import configRedata from './redata';
 import reactRedata from './react/react-redata';
 
-export default redata;
+export default configRedata;
```

Comment:
I tried to keep a consistency between what we're exporting here and what we're importing. On redata file what we're really exporting by default is the config redata fn, and we're here importing it as 'redata'. I don't have a strong opinion on this but I felt like there were too many  `redata` keywords around the code with slight different meanings..